test/functional/p2p_erlay.py

```diff
@@ -0,0 +1,503 @@
+#!/usr/bin/env python3
+# Copyhigh (c) 2016-2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test reconciliation-based transaction relay protocol."""
+
+from io import BytesIO
+from test_framework.siphash import siphash256
+import random
+import struct
+import time
+
+from test_framework.key import TaggedHash
+from test_framework.messages import (
+    msg_inv, msg_getdata, msg_wtxidrelay,
+    msg_verack, msg_sendrecon, msg_reqrecon,
+    msg_sketch, msg_reqsketchext, msg_reconcildiff,
+    MSG_WTX, CTransaction, CInv,
+)
+from test_framework.p2p import P2PDataStore, MIN_P2P_VERSION_SUPPORTED
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal, hex_str_to_bytes
+
+# These parameters are specified in the BIP-0330.
+Q_PRECISION = (2 << 14) - 1
+FIELD_BITS = 32
+FIELD_MODULUS = (1 << FIELD_BITS) + 0b10001101
+
+# These parameters are suggested by the Erlay paper based on analysis and simulations.
+DEFAULT_Q = 0.02
+RECON_REQUEST_INTERVAL = 16
+INBOUND_PENALIZED_INVENTORY_BROADCAST_INTERVAL = 5
+MAX_OUTBOUND_FLOOD_TO = 8
+
+BYTES_PER_SKETCH_CAPACITY = FIELD_BITS / 8
+
+#################################
+######## General helpers ########
```

Comment:
[The linter](https://cirrus-ci.com/task/6704150097428480) is not happy with this comment style.

Reply:
Not only with that.  I ran pycodestyle and got many warnings about it.

![image](https://user-images.githubusercontent.com/19480819/115130588-3c113100-9fc7-11eb-8efd-77b0b406f732.png)

Reply:
@brunoerg I think following best practices beyond what *our* linter does is not necessary, but still I tried to resolve most of those complaints from `pycodestyle` where it makes sense.